src/storage/hummock_sdk/src/table_watermark.rs

```diff
@@ -44,12 +44,15 @@ pub struct TableWatermarksIndex {
     // later epoch at the back
     pub staging_watermarks: VecDeque<(HummockEpoch, Arc<[VnodeWatermark]>)>,
     pub committed_watermarks: Option<Arc<TableWatermarks>>,
-    latest_epoch: HummockEpoch,
-    committed_epoch: HummockEpoch,
+    latest_epoch: Option<HummockEpoch>,
+    committed_epoch: Option<HummockEpoch>,
```

Comment:
In which case will these two be None?

Reply:
Previously, when an empty `WatermarkIndex` is created, the `latest_epoch` is None, but now I just realize that the `latest_epoch` can always be `Some(_)`, because all `WatermarkIndex::new` is followed by a `add_epoch_watermark`, so I change the `new` method to create the `WatermarkIndex` together with the first watermark, and then the `latest_epoch` can always be `Some(_)` and we can remove the `Option`.

The `committed_epoch` is `None` when a new watermark is written, but no data has ever been committed for this table, so the `committed_epoch` of this table id is `None`.